scripts/generate_declaration_export_for_ccs_sourcing.py

```diff
@@ -0,0 +1,271 @@
+"""
+
+Usage:
+    scripts/generate_declaration_export_for_ccs_sourcing.py <data_api_url> <data_api_token> <path_to_declaration_manifest> # noqa
```

Comment:
Amazingly, adding `#noqa` at the end of this line prevents the script from running.

``
$ python scripts/generate_declaration_export_for_ccs_sourcing.py http://localhost:5000 myToken ../digitalmarketplace-supplier-frontend/app/content/frameworks/g-cloud-7/manifests/declaration.yml
Usage:
    scripts/generate_declaration_export_for_ccs_sourcing.py <data_api_url> <data_api_token> <path_to_declaration_manifest> # noqa
``